rust/main/Cargo.toml

```diff
@@ -126,24 +126,9 @@ sha2 = { version = "0.10.6", default-features = false }
 sha256 = "1.1.4"
 sha3 = "0.10"
 solana-account-decoder = "=1.14.13"
-solana-banks-client = "=1.14.13"
```

Comment:
were you able to remove all these because they're specified down below in patch?

Reply:
`solana-banks-client` is not used in `main` workspace and it is not specified in `patch` section of this file.